src/components/SongDetail.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { connect } from "react-redux";
+
+const SongDetail = ({ selectedSong }) => {
+  if (!selectedSong) {
```

Comment:
można, ale lepiej wszystko w returnie ostatnim określać, czyli tam to uwarunkować zamiast nad nim

Reply:
przykładowo:
```

return (
 <>
 {selectedSong 
    ? 
      <div>
       <div>Title: {selectedSong.title}</div>
       <div>Duration: {selectedSong.duration}</div>
       </div>
   :  <div>Select a song</div>}
 </>

```
)